docs/pages/enroll-resources/desktop-access/troubleshooting.mdx

```diff
@@ -393,3 +393,34 @@ to force the policy update:
 ```code
 $ gpupdate.exe /force
 ```
+### Shared Directory Error
+
+Attempts to share directory on a remote desktop session results in the error:
+
+```
+Encountered shared directory error 1
+```
```

Comment:
What do you think about moving this configuration to the setup guide instead of mentioning this poor error message here in the troubleshooting guide?

We can file a separate issue to improve the error message as well.